ui/src/Explore/CommitHeader.tsx

```diff
@@ -16,43 +16,24 @@
 
 import {Card} from "react-bootstrap";
 import prettyMilliseconds from "pretty-ms";
-import React, {useEffect, useState} from "react";
-import {api, CommitMeta} from "../utils";
+import React from "react";
+import {CommitMeta} from "../utils";
 
-function fetchLog(currentRef: string, setLog: (v: CommitMeta) => void) {
-  return api().getCommitLog({'ref': currentRef})
-    .then((data) => {
-      if(data.operations && data.operations.length > 0) {
-        setLog(data.operations[0]);
-      }
-    }).catch(t => console.log(t));
-}
-
-function CommitHeader(props: {currentRef: string}) {
-
-  const [currentLog, setLog] = useState<CommitMeta>();
+function CommitHeader({committer, author, hash, message, properties, commitTime}: CommitMeta) {
```

Comment:
WDYT about using `CommitMeta` as the only arg to simplify parameter passing to this method?

Reply:
It has no effect on parameter passing. You can only pass basic types through these react components. This is syntactic sugar to avoid having to reference each element of the commit log

Reply:
TIL :+1: